bin/cdk-assignment.ts

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
+import { devStackProps } from '../configuration_parameters'
```

Comment:
I think devEnv will be a better name then devStack as dev is an environment name